lib/src/Uncertainty/Algorithm/OrthogonalBasis/OrthogonalProductFunctionFactory.cxx

```diff
@@ -86,6 +87,13 @@ EnumerateFunction OrthogonalProductFunctionFactory::getEnumerateFunction() const
   return tensorizedFunctionFactory_.getEnumerateFunction();
 }
 
+/* Return the collection of univariate orthogonal polynomial families */
+OrthogonalProductFunctionFactory::FunctionFamilyCollection OrthogonalProductFunctionFactory::getFunctionFamilyCollection() const
+{
+  return coll_;
```

Comment:
Can't you just call `tensorizedFunctionFactory_.getFunctionFamilyCollection()`?
I dislike having coll both in this class and in tensorizedFunctionFactory_.